CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Read "lean-toolchain" files and do not set --default-toolchain to something specific.
 - elan should not install multiple toolchains.
+- Occurrence highlighting can be configured to ignore string matches and use only Lean's LSP responses
```

Comment:
As previously discussed, you can leave this out because we don't really update `CHANGELOG.md` anymore.

Reply:
Done. Should we delete the file?

Reply:
Yes, I think it's fine to delete it. I don't think the VS Code marketplace uses it and given how incomplete it is, it can't be particularly helpful to anyone.